src/mame/apple/apple2e.cpp

```diff
@@ -2776,12 +2786,17 @@ void apple2e_state::update_iic_mouse()
 			if (m_xy)
 			{
 				m_yirq = true;
-				raise_irq(IRQ_MOUSEXY);
+				raise_mousexy_irq = true;
 			}
 		}
 
 		m_y0 = !m_y0;
 	}
+
+	if (raise_mousexy_irq) {
+		raise_irq(IRQ_MOUSEXY);
+		raise_mousexy_irq = false;
+	}
```

Comment:
There’s no point clearing the local variable `raise_mousexy_irq` when it won’t be used again.  Also, this file uses Allman brace positioning – please follow the example of the existing code.

Reply:
Will change.